ci/lint.sh

```diff
@@ -71,6 +93,14 @@ if [ "$LINT" ]; then
             RET=1
         fi
     done
+
+    trailing_space_pxd=$(find ../**/*.pxd -type f -exec egrep -l " +$" {} \;)
+    if [[ -n $trailing_space_pxd ]]
+    then
+        RET=1
+        echo $trailing_space_pxd
+    fi
```

Comment:
This looks great, thanks.  Any other ideas you have for linting cython files will be a big hit with the maintainers (separate PR(s))

Reply:
Sounds great! I am not sure if I'll have much time in the next little bit after this gets merged but I'll do my best.